extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -18,19 +18,17 @@ syntax = "proto3";
 package stackdriver.config.v1alpha1;
 
 message PluginConfig {
-  // Indicates whether the module should act as inbound or outbound reporter.
-  // Difference being:
-  //   * inbound and outbound have different metric types to export.
-  //   * inbound reporter writes access logs.
-  enum ReporterKind {
-    INBOUND = 0;
-    OUTBOUND = 1;
-  }
-  ReporterKind kind = 1;
-
   // Controls whether to export access log.
-  bool disable_access_logging = 2;
+  bool disable_access_logging = 1;
 
   // FQDN of destination service that the request routed to.
-  string destination_service_name = 3;
+  string destination_service_name = 2;
+
+  // The endpoint that plugin targets for metric reporting. If not specified,
+  // the default Stackdriver monitoring endpoint will be used.
+  string monitoring_endpoint = 3;
```

Comment:
just curious: how are we doing auth for non-default endpoints?

Reply:
Opencensus library uses default GCP credential.

Reply:
I'd recommended labelling it as such, e.g. `test_monitoring_endpoint`.

Reply:
Done.